evaluators/embedding/L1NormEvaluator/tests/test_l1norm.py

```diff
@@ -0,0 +1,34 @@
+from math import sqrt
+
+import numpy as np
+import pytest
+
+from .. import L1NormEvaluator
+
+
+@pytest.mark.parametrize(
+    'doc_embedding, gt_embedding, expected',
+    [
+        ([0, 0], [0, 0], 0.0),
+        ([0, 0], [0, 1], 1.0),
+        ([0, 0], [1, 0], 1.0),
+        ([0, 0], [1, 1], 2.0),
+        ([0, 1], [0, 0], 1.0),
+        ([0, 1], [0, 1], 0.0),
+        ([0, 1], [1, 0], 2.0),
+        ([0, 1], [1, 1], 1.0),
+        ([1, 0], [0, 0], 1.0),
+        ([1, 0], [0, 1], 2.0),
+        ([1, 0], [1, 0], 0.0),
+        ([1, 0], [1, 1], 1.0),
+        ([1, 1], [0, 0], 2.0),
```

Comment:
Check what happens if 2 embedding of different dimensions arrive

Reply:
Added different dimensions